.github/workflows/go.yml

```
@@ -0,0 +1,23 @@
+name: Go
+
+on: [push]
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: github.com/kohofinancial/errors
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+
+      # On this stage, we need the path to be the Github repository path for proper stack traces.
+      - name: Test
+        run: |
+          cd github.com/kohofinancial/errors
+          go test -v ./...
```

If you need the path (I don't think you do), you could probably shorten to

```suggestion
          go test -v github.com/kohofinancial/errors/...
```

---

Opted to add a default path for run instead. Didn't seem to be a way to do it globally for all steps though.